content/doc/book/managing/plugins.adoc

```diff
@@ -182,7 +188,7 @@ plugins, and present a button for doing so.
 image::managing/plugin-manager-uninstall.png["Installed tab in the Plugin Manager", role=center]
 
 
-A plugin may also be uninstalled by removing the corresponding `.hpi`
+A plugin may also be uninstalled by removing the corresponding `.jpi`
 file from the `JENKINS_HOME/plugins` directory on the controller. The plugin will
 continue to function until the controller has been restarted.
```

Comment:
Could you also change the "If a plugin `.hpi` file is removed but required by other plugins" part below this?